climada/hazard/tc_tracks.py

```diff
@@ -2435,3 +2435,18 @@ def set_category(max_sus_wind, wind_unit='kn', saffir_scale=None):
         return (np.argwhere(max_wind < saffir_scale) - 1)[0][0]
     except IndexError:
         return -1
+
+def _zlib_from_DataArray(data_var: xr.DataArray) -> bool:
```

Comment:
Linter dislikes uppercase in function names
```suggestion
def _zlib_from_dataarray(data_var: xr.DataArray) -> bool:
```